### PR TITLE
ast.If

### PR DESCRIPTION
**Pull request by jkarp314**
add imports in ast.If nodes to the list of questionable imports

@CJ-Wright

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -77,6 +77,17 @@ def __init__(self):
         from .relative_function_inside_class import a_third_function
         import pprint
 """},
+
+{'targets':
+     {'questionable': ['groucho', 'harpo', 'chico']},
```
Comment (CJ-Wright) [inline test.py:82]: I would update the ordering of the questionable elements or change the tests to only check that the sets are equivalent. I'm not certain that we care about the ordering of the deps in depfinder at the moment, so laundering them through set might be ok.

Comment (jkarp314) [reply test.py]: I changed the order and now it works fine

**Comment by CJ-Wright** [comment]
@ericdill thoughts?

**Review by ericdill — APPROVED** [review]
Lgtm! thx @CJ-Wright @jkarp314

Merge is in your hands whenever you're happy @CJ-Wright

**Pull request merged**